draft-ietf-trans-rfc6962-bis.md

```diff
@@ -784,10 +782,13 @@ extension that it does not understand, it SHOULD ignore that extension.
 Furthermore, an implementation MAY choose to ignore any extension(s) that it
 does understand.
 
-The encoding of the digitally-signed element is defined in [RFC5246].
+`algorithm` indicates the signature scheme used to compute the signature.  The
+SignatureScheme enumeration is defined in {{!I-D.ietf-tls-tls13}}.
```

Comment:
For consistency, could we stick to using just explicit references (in the "normative:" and "informative:" sections)?